packages/lpa-questionnaire-web-app/tests/unit/lib/check-answers-sections.test.js

```diff
@@ -0,0 +1,131 @@
+const checkAnswersSections = require('../../../src/lib/check-answers-sections');
+
+jest.mock('../../../src/services/task.service', () => ({
+  __esModule: true,
+  SECTIONS: [
+    {
+      sectionId: 'mockSection',
+      tasks: [
+        {
+          taskId: 'mockTask1',
+          href: '/mock-task-1',
+          rule: () => 'NOT STARTED',
+        },
+      ],
+    },
+  ],
+  HEADERS: {
+    mockSection: 'Mock Section',
+    mockTask1: 'Mock Task 1',
+  },
+}));
+
+describe('check-answers-sections', () => {
+  [
+    {
+      title: 'Shows no answer found if no results',
+      taskValue: null,
+      output: { text: 'No answer found' },
+      showActions: undefined,
+    },
+    {
+      title: 'Returns no files uploaded if uploaded files key present but empty',
+      taskValue: {
+        uploadedFiles: [],
+      },
+      output: { html: '<p>No files uploaded</p>' },
+    },
+    {
+      title: 'Shows file list if uploaded files key present and not empty',
+      taskValue: {
+        uploadedFiles: [{ name: 'mock-file.pdf', id: 'abc-123' }],
+      },
+      output: { html: '<ul class="govuk-list"><li>mock-file.pdf</li></ul>' },
+    },
+    {
+      title: 'Returns empty string if value of field is empty string',
+      taskValue: {
+        mockValue: '',
+      },
+      output: { html: '' },
+    },
+    {
+      title: 'Shows Yes if boolean value true provided',
+      taskValue: {
+        mockValue: true,
+      },
+      output: { html: '<p>Yes</p>' },
+    },
+    {
+      title: 'Shows No if boolean value false provided',
+      taskValue: {
+        mockValue: false,
+      },
+      output: { html: '<p>No</p>' },
+    },
+    {
+      title: 'Shows value if string is provided',
+      taskValue: {
+        mockValue: 'mock value',
+      },
+      output: { html: '<p>mock value</p>' },
+    },
+  ].forEach(({ title, taskValue, output }) => {
```

Comment:
This doesn't destructure showActions on line 29.

Reply:
Fixed as removed reference - vestige of approach that didn't work